libr/util/punycode.c

```diff
@@ -15,7 +15,7 @@ ut32 adapt_bias(ut32 delta, unsigned n_points, int is_first) {
 	delta /= is_first ? DAMP : 2;
 	delta += delta / n_points;
 
-	for (k = 0; k > ((BASE - TMIN) * TMAX) / 2; k += BASE) {
+	for (k = 0; delta > ((BASE - TMIN) * TMAX) / 2; k += BASE) {
```

Comment:
I think this `for` doesn't have much sense, since there is `k` that is initialised and incremented, but it's actually never used in the body of the for nor in the condition